src/client/src/commons/form/profile/components/profileLayers/components/chronoStratigraphyLayers/chronostratigraphyPanel.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from "react";
+import { Stack, Typography, CircularProgress, Box } from "@mui/material";
+import LayerCard from "./layerCard";
+import LayerGap from "./layerGap";
+import { useTranslation } from "react-i18next";
+import ChronostratigraphyLayers from "./chronostratigraphyLayers";
+import LithologyLayers from "./lithologyLayers";
+import { useChronostratigraphies } from "../../../../../../../api/fetchApiV2";
+
+const getHeader = t => {
+  return [
+    { title: t("eon"), isVisible: true },
+    { title: t("era"), isVisible: true },
+    { title: t("period"), isVisible: true },
+    { title: t("epoch"), isVisible: true },
+    { title: t("subepoch"), isVisible: true },
+    { title: t("age"), isVisible: true },
+    { title: t("subage"), isVisible: true },
+  ];
+};
+
+const ChronostratigraphyPanel = ({ selectedStratigraphyID, isEditable }) => {
+  const { data: chronostratigraphyQueryData } = useChronostratigraphies(
+    selectedStratigraphyID,
+  );
+  const { t } = useTranslation();
+
+  const [header, setHeader] = useState(getHeader(t));
+  const [navigationState, setNavigationState] = useState(null);
+
+  useEffect(() => {
+    return setHeader(getHeader(t));
+  }, [t]);
+
+  const layerDisplayStack = [];
```

Comment:
Q: brauchts den layerDisplayStack in dieser Komponente auch, so wie in chronostratigraphyLayers? Falls ja, kann man es auslagern und wiederverwenden?

Reply:
Braucht es hier nicht, ist entfernt.